src/sentry/conf/server.py

```diff
@@ -1381,6 +1381,16 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 CRISPY_TEMPLATE_PACK = "bootstrap3"
 # Sentry and internal client configuration
 
+
+SENTRY_FEATURES_DESCRIPTIONS = {
```

Comment:
So even if a feature requires additional infra, it'll be in here? What do you think is the best way to communicate how to set up this infra? Usually it's a few docker containers in the `docker-compose.yml` file. We could potentially add some documentation around features [here](https://develop.sentry.dev/self-hosted/) in our develop documentation that describes this

Reply:
I agree with Hubert on this one (and obviously I'm not an employee). Most of new features will require some new containers (either Snuba ingester or Sentry worker), or even a new Kafka topic creation (in which we should do all that manually on self-hosted since the Kafka does not support [automatic topic creation](https://kafka.apache.org/documentation/#brokerconfigs_auto.create.topics.enable) yet). So, having EA feature flag toggle here most likely be not viable enough. But at the same time, if we're pushing all the containers on self-hosted just for enabling new features, and having them being idle will make some complaints for the resources that it took.

I think it's best if we create a new CLI scripts on self-hosted that generates `docker-compose.override.yml` consisting of the containers we'll actually use for enabling some new features.

But that will bring us into a chaotic configuration as we're using [YAML anchoring a lot](https://github.com/getsentry/self-hosted/blob/bd1d1d822ad1d676870ea2160213788af7e21d3f/docker-compose.yml#L1-L101).